src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,39 @@
 import React from 'react';
+import { Comments } from '../../types/comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comments;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <>
+      <div className="CommentInfo__title">
+        <strong
+          className="CommentInfo__name"
+        >
+          {name}
+        </strong>
+
+        {' by '}
```

Comment:
Wrap in span and remove expression, fix in all cases

Reply:
done